### PR TITLE
Fix running eyes in circle via "test eyes" flag

### PR DESCRIPTION
**Pull request by davidsbailey**
The `[test eyes]` commit flag has been unusable lately because it causes circle to run out of memory (8G). The out of memory errors ([example](https://circleci.com/gh/code-dot-org/code-dot-org/35271)) give very helpful error messages pointing to a [breakdown of memory usage](https://35271-24250009-gh.circle-artifacts.com/1/tmp/memory-usage.txt), which show **900MB used per unicorn worker**. 

Reducing the number of workers from 7 to 4 does not appear to have any negative impact on circle performance or stability, and allows eyes tests to be run in parallel with UI tests without running out of memory. Therefore, reduce the number of workers to 4.

Enabling eyes tests increases Circle container 1 run time by 12 minutes from approximately 24:00 to 36:00. Therefore, keep eyes tests disabled unless the `[test eyes]` flag is specified.

### CODE REVIEW

**Comment by davidsbailey** [comment]
FYI @balderdash that (1) this may affect circle performance, even though my few test runs say it doesn't and (2) ~~my inclusion of the `[test eyes]` flag in the PR title may slightly throw off your stats when it causes eyes tests to be run in the staging circle branch.~~ EDIT: I've changed the PR title so that it doesn't run eyes on the staging circle run.

**Comment by davidsbailey** [comment]
@islemaster it's worth noting that `[test eyes]` currently causes eyes tests to be run without actually looking at eyes diffs. There are still some other hurdles to get circle to actually look at the eyes diffs, which I propose enabling behind a different commit message flag, e.g. `[test eyes diffs]`.

File: dashboard/test/ui/features/pd/workshopCertificates.feature [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 Feature: Basic appearance for Workshop Certificates
 
+@no_circle
```
Comment (islemaster) [inline dashboard/test/ui/features/pd/workshopCertificates.feature:6]: Disabling this scenario on Circle wasn't mentioned in your PR description.  What's the reason behind this?

Comment (davidsbailey) [reply dashboard/test/ui/features/pd/workshopCertificates.feature:6]: This scenario failed in an earlier circle run: https://circleci.com/gh/code-dot-org/code-dot-org/35586

**Pull request merged**